Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -219,7 +219,7 @@
     if (path.startsWith('/'))
         path = path.substring(1);
 
-    auto *cocoaPath = static_cast<NSString *>(path);
+    RetainPtr cocoaPath = path.createNSString();
```

Comment:
All of the `WebKit/*/Extensions` files are ARC and these are not leaks.

I would prefer to keep the casts as-is, or front load the `.get()`.
```suggestion
    auto *cocoaPath = path.createNSString().get();
```

Reply:
I am not trying to fix leaks. Darin said in his review that he preferred we call a `createNSString()` function rather than using a cast. It also has the benefit of not using autorelease unnecessarily. That said, I can totally use `auto *cocoaPath = path.createNSString().get();` like you suggest since this code uses ARC.